pkg/jira/jira.go

```diff
@@ -83,7 +86,7 @@ func (c *Verifier) ghUnlabeledPRs(extPR pr) ([]pr, error) {
 
 func (c *Verifier) verifyExtPRs(issue *jiraBaseClient.Issue, extPRs []pr, errs []error, tagName string) (bool, string, []error, bool) {
 	var success bool
-	message := fmt.Sprintf("Jira: fix included in accepted release %s", tagName)
+	message := fmt.Sprintf("Bugfix included in accepted release %s", tagName)
```

Comment:
Nit... should this be "Jira Issue" or just "Issue" instead of anything related to "bug"?